.github/workflows/validate_company_industry.yml

```diff
@@ -34,7 +35,8 @@ jobs:
       - name: Get changed files
         id: changed_files
         run: |
-          echo "::set-output name=files::$(git diff --name-only ${{ github.event.before }} ${{ github.sha }})"
+          changed_files=$(git log --name-only $(git merge-base origin/main HEAD)..HEAD | grep priv/companies/)
```

Comment:
TY to @onno-vos-dev for this one git liner!

Reply:
We went with a published action instead of home grown for this @onno-vos-dev . see https://github.com/beam-community/elixir-companies/pull/717#discussion_r1823273147 if you are curious. TY again for you assistance on this.